LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -335,7 +341,7 @@ struct SectionWithTapToEdit<Content, NavigationDestination>: View where Content:
     let addExtraSpaceAboveSection: Bool
     let title: String
     let descriptiveText: String
-    let destination: NavigationDestination
+    let destination: (@escaping () -> Void) -> NavigationDestination
```

Comment:
For clarity's sake, can you add a parameter name to the return type here?
i.e. `(_ goBack: @escaping () -> Void) -> NavigationDestination`